nimble/host/mesh/src/prov.c

```diff
@@ -166,6 +167,7 @@ struct prov_rx {
 #define RETRANSMIT_TIMEOUT   K_MSEC(500)
 #define BUF_TIMEOUT          K_MSEC(400)
 #define TRANSACTION_TIMEOUT  K_SECONDS(30)
+#define PROTOCOL_TIMER       os_cputime_usecs_to_ticks(60000000)
```

Comment:
`ble_npl_time_ms_to_ticks32`
and not really sure how this could even work since cputime time is 1/32768s while os tick (which callouts use) is 1/128s...

Reply:
Done

Reply:
Done